DFC.ServiceTaxonomy.ApiFunction/Function/Execute.cs

```diff
@@ -64,7 +64,8 @@ public Execute(IOptionsMonitor<ServiceTaxonomyApiSettings> serviceTaxonomyApiSet
 
                 //Add the host in to all cypher queries
 
-                cypherQueryParameters.Add("host", BuildHostFunctionUrl(req, log));
+                cypherQueryParameters.Add("apiHost", BuildHostFunctionUrl(req, log));
+                cypherQueryParameters.Add("websiteHost", $"{_serviceTaxonomyApiSettings.CurrentValue.Scheme}{_serviceTaxonomyApiSettings.CurrentValue.WebsiteHost}");
```

Comment:
Doesn't the scheme represent the scheme of the API endpoint, rather than the scheme of the website?

Reply:
Yes it does, I can refactor a different scheme for each just didn't want to create settings galore and would need DevOps to refactor the original scheme too